nuxeo-core/nuxeo-core-storage-dbs/src/main/java/org/nuxeo/ecm/core/storage/dbs/DBSDocumentState.java

```diff
@@ -48,6 +50,10 @@
      */
     protected State originalState;
 
+    private String id;
+
+    private String parentId = UNDEFINED;
```

Comment:
You might want to affect UNDEFINED_PARENT_ID instead and not rely on javax constant.